Arm/rover_arm_system.hpp

```diff
@@ -191,42 +187,96 @@ class RoverArmSystem : public sjsu::common::RoverSystem
                 float pointer,
                 float middle,
                 float ring,
-                float pinky)
+                float pinky,
+                float pitch,
+                float roll)
```

Comment:
Same as mentioned earlier, will need to create a struct to make this cleaner and best practice